ci/devices.json

```diff
@@ -5,7 +5,7 @@
     "board": "SLIMEVR"
   },
   {
-    "platform": "espressif32 @ 3.5.0",
+    "platform": "espressif32 @ ^4.4.0",
```

Comment:
```suggestion
    "platform": "espressif32 @ 4.4.0",
```

Reply:
since platformio doesn't have a lockfile, I think we need to use the exact version as Anna has suggested